From 9df022bd38f7eea9b7a693ea1101611d19127189 Mon Sep 17 00:00:00 2001
From: Diego Pino Garcia <dpino@igalia.com>
Date: Mon, 22 May 2023 19:58:50 -0700
Subject: [PATCH] Cherry-pick 264395@main (4977290ab4ab).
 https://bugs.webkit.org/show_bug.cgi?id=256917

    [GLIB] Fix build error after 264196@main
    https://bugs.webkit.org/show_bug.cgi?id=256917

    Reviewed by Michael Catanzaro.

    Variable BWRAP_EXECUTABLE is only defined when BUBBLEWRAP_SANDBOX is
    enabled.

    * Source/WTF/wtf/glib/Sandbox.cpp:
    (WTF::isInsideUnsupportedContainer):

    Canonical link: https://commits.webkit.org/264395@main

Canonical link: https://commits.webkit.org/260527.371@webkitglib/2.40
---
 Source/WTF/wtf/glib/Sandbox.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Source/WTF/wtf/glib/Sandbox.cpp b/Source/WTF/wtf/glib/Sandbox.cpp
index 7d84e830ab33..9b07bb8cb5a9 100644
--- a/Source/WTF/wtf/glib/Sandbox.cpp
+++ b/Source/WTF/wtf/glib/Sandbox.cpp
@@ -36,6 +36,7 @@ bool isInsideFlatpak()
     return returnValue;
 }
 
+#if ENABLE(BUBBLEWRAP_SANDBOX)
 bool isInsideUnsupportedContainer()
 {
     static bool inContainer = g_file_test("/run/.containerenv", G_FILE_TEST_EXISTS);
@@ -64,6 +65,7 @@ bool isInsideUnsupportedContainer()
 
     return inContainer && !supportedContainer;
 }
+#endif
 
 bool isInsideSnap()
 {
-- 
2.39.3

